src/util/ChartUtils.ts

```diff
@@ -734,25 +734,19 @@ export function getTooltipNameProp(
 export function inRange(
   x: number,
   y: number,
-  scale: number,
```

Comment:
This might break the calculations when someone is zoomed in?

Reply:
Nvm you do this later

Reply:
Technically earlier 😅 the coordinates arrive to this function already scaled. The storybook with zoom in/out was useful.